pulumi/infra/config.py

```diff
@@ -120,6 +120,8 @@ def _require_env_var(key: str) -> str:
 
 
 OPERATIONAL_ALARMS_EMAIL = _require_env_var("GRAPL_OPERATIONAL_ALARMS_EMAIL")
+REDIS_PORT = _require_env_var("REDIS_PORT")
+KAFKA_BROKER_PORT = _require_env_var("KAFKA_BROKER_PORT")
```

Comment:
Shower thoughts:

We may want to rejigger this into an endpoint, and for local make it like 
LOCALHOST_SENTINEL_VALUE:6379
and have Nomad do a substitute